### ISSUE TITLE
Help producing results with the StyleGAN models

### ISSUE THREAD

**Issue by james-oldfield** [lukemelas/unsupervised-image-segmentation#4]
Hi there!

I'm having trouble producing meaningful results on StyleGAN2 on AFHQ. I've been using the default setup and hyperparameters. After 50 iterations (with the default batch size of 32) I get visualisations that look initially promising: (https://i.imgur.com/eR79Wyd.png). But as training progresses, and indeed when it reaches 300 iterations, these are the visualisation results: https://i.imgur.com/36zhBzT.png.

I've tried playing with the learning rate, and the number of iterations with no success yet. Did you have tips here or ideas as to what might be going wrong here?

Thanks!
James.

**Comment by james-oldfield** [comment]
I've just seen from the ICLR version of the paper's sup. material that a 0.2 radius is useful for StyleGAN, which indeed helps a little bit (100 iterations is not too bad: https://i.imgur.com/kzPgzuJ.png). If you have any other tips here to produce more faithful results please do let me know!
